src/validations/index.js

```diff
@@ -1,4 +1,11 @@
+var i18n = require('i18next-client');
+i18n.init({
```

Comment:
@kinduff great question—I would say `application.js` is probably the right place for this, actually. As things get more complex we’ll probably need some sort of `initializers` structure but for now `application.js` would be the place for “general setup of the environment”